CHANGES.md

```diff
@@ -24,6 +24,19 @@ OpenSSL 3.1
 
 ### Changes between 3.0 and 3.1.0 [xx XXX xxxx]
 
+ * Our provider implementations of `OSSL_FUNC_KEYMGMT_EXPORT` and
+   `OSSL_FUNC_KEYMGMT_GET_PARAMS` for EC and SM2 keys now honor
+   `OSSL_PKEY_PARAM_EC_POINT_CONVERSION_FORMAT` as set (and
+   default to `POINT_CONVERSION_UNCOMPRESSED`) when exporting
+   `OSSL_PKEY_PARAM_PUB_KEY`.
+   For symmetry, our implementation of `EVP_PKEY_ASN1_METHOD->export_to`
```

Comment:
I would mention explicitly that this means a change of the default format from compressed to uncompressed here, and that this may cause issues for applications which expect compressed output.

Reply:
I amended the CHANGES entry based on your feedback with fc0288cbe250c741deb552fa21522125990a313a.

It is important to note that we never mandated any specific choice of the point conversion formats in the documentation for the interface nor for our implementation, and we always specified that any of the SECG standard point encodings was supported.

Applications expecting "COMPRESSED" output from a keymgmt export/to_data/get_params operations were already non-complying and potentially incompatible with 3rd party providers or future versions of OpenSSL providers.
If this PR is merged, it does not change the fact that applications and providers should properly parse the octet string retrieved from the export operation to decode it according to SECG definitions.